project/src/main/java/com/google/sps/servlets/PollingLocationsServlet.java

```diff
@@ -0,0 +1,34 @@
+package com/google/sps/servlets;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.sps.infocompiler;
+
+private static InfoCompiler infoCompiler = new InfoCompiler();
+private final static String VOTER_INFO_QUERY_URL =
+    String.format("https://www.googleapis.com/civicinfo/v2/voterinfo?key=%s",
+                  Config.CIVIC_INFO_API_KEY);
+@WebServlet("/pollingLocation")
+public class PollingLocationServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
+    String userAddress = request.getParameter("address");
+    String queryUrl = String.format("%s&address=%s", VOTER_INFO_QUERY_URL, userAddress);
+    JsonArray voterInfoArray = infoCompiler.queryCivicInformation(queryUrl);
+    if (voterInfoArray.has("pollingLocations")) {
+      JsonArray pollingLocation voterInfoArray.getAsJsonArray("pollingLocations");
+    }
+    JsonObject pollingLocationAddress pollingLocationsArray.get(0).getAsJsonObject();
```

Comment:
- Assign the value
- Did you mean to use `pollingLocation` instead of `pollingLocationArray`?